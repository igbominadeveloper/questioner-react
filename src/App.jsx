import React from 'react';
import { Route, Switch } from 'react-router-dom';

//components
import Signup from './components/containers/Signup/Signup';
import Login from './components/containers/Login/Login';
import Homepage from './pages/Landing';
import NewMeetup from './components/containers/NewMeetup/NewMeetup';

const App = () => {
  return (
    <Switch>
      <Route exact path="/" component={Homepage} />
      <Route path="/signup" component={Signup} />
      <Route path="/login" component={Login} />
      <Route path="/organize" component={NewMeetup} />
    </Switch>
  );
};

export default App;
