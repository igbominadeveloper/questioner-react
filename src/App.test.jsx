import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
  describe('Render', () => {
    it('renders without error', () => {
      const component = shallow(<App />);
      expect(component).toMatchSnapshot();
    });
  });
});
