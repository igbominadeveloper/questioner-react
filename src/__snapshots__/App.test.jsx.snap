// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Render renders without error 1`] = `
<Switch>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(HomePage)",
        "type": [Function],
      }
    }
    exact={true}
    path="/"
  />
  <ConnectFunction
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Signup)",
        "type": [Function],
      }
    }
    path="/signup"
  />
  <ConnectFunction
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Login)",
        "type": [Function],
      }
    }
    path="/login"
  />
  <withRouter(AdminWrapper)
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(NewMeetup)",
        "type": [Function],
      }
    }
    path="/organize"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(SingleMeetup)",
        "type": [Function],
      }
    }
    exact={true}
    path="/meetup=:meetupId"
  />
</Switch>
`;
